opfpush-providers/gcm/src/main/java/org/onepf/opfpush/gcm/GCMPubSubHelper.java

```diff
@@ -131,7 +159,14 @@ private boolean checkPubSubAvailability(@NonNull final OPFPushHelper helper, @No
         return true;
     }
 
-    public class CallbackMainThreadWrapper implements Callback {
+    public interface Callback {
```

Comment:
This interface should be used only for GCMPubSubHelper methods therefore it is inner interface. There are a lot of inner Callback interfaces in the android SDK.